metadata/service/server/single.yml

```diff
@@ -0,0 +1,10 @@
+applications:
+- watchdog
+classes:
+- service.watchdog.support
+parameters:
+  watchdog:
+    server:
+      enabled: true
+      bind:
```

Comment:
Watchdog doesn't bind, this is not needed

Reply:
Fixed...